dspace-xmlui/src/main/java/org/dspace/app/xmlui/aspect/submission/submit/UploadWithEmbargoStep.java

```diff
@@ -262,7 +262,8 @@ else if(editPolicy!=null && isAdvancedFormEnabled){
         //  If the user has already uploaded files provide a list for the user.
         if (bitstreams.size() > 0 || disableFileEditing)
 		{
-	        Table summary = div.addTable("submit-upload-summary",(bitstreams.size() * 2) + 2,7);
+            int cols= disableFileEditing?6:7;
```

Comment:
Also, not related directly to this PR, but there doesn't seem to be a common way of using indentation (spaces or tabs?) and placement of opening braces?